tools/config_updater.py

```diff
@@ -100,15 +105,39 @@
 parser_set_var.add_argument("var", help="Variable to set.")
 parser_set_var.add_argument("val", help="Value to set.")
 
+def AddUser(username, password, labels=[], token=None):
+  """Implementation of the add_user command."""
+  with aff4.FACTORY.Create("aff4:/users/%s" % username,
+                           "GRRUser", mode="rw", token=token) as fd:
+    # Note this accepts blank passwords as valid.
+    if not password:
+      password = getpass.getpass(
+          prompt="Please enter password for user '%s': " % username)
+    fd.SetPassword(password)
+
+    # Use sets to dedup input.
+    if labels is not []:
```

Comment:
This whole block can be done as
if labels:
  fd.AddLabels(*labels, owner="GRRR")